plugins/woocommerce-admin/client/marketing/overview-multichannel/IntroductionBanner/IntroductionBanner.tsx

```diff
@@ -0,0 +1,165 @@
+	 * that will do nothing upon click.
+	 *
+	 * If there are registered channels and recommended channels,
+	 * the Channels card will display the  "Add channels" toggle button,
+	 * and clicking on the "Add channels" button in this introduction banner
+	 * will scroll to the button in Channels card.
+	 */
+	const showAddChannelsButton =
+		!! dataRegistered?.length && !! dataRecommended?.length;
+
+	return (
+		<Card className="woocommerce-marketing-introduction-banner">
+			<div className="woocommerce-marketing-introduction-banner-content">
+				<div className="woocommerce-marketing-introduction-banner-title">
+					{ __(
+						'Reach new customers and increase sales without leaving WooCommerce',
+						'woocommerce'
+					) }
+				</div>
+				<Flex
+					className="woocommerce-marketing-introduction-banner-features"
+					direction="column"
+					gap={ 1 }
+					expanded={ false }
+				>
+					<FlexItem>
+						<Flex>
+							<Icon icon={ trendingUp } />
+							<FlexBlock>
+								{ __(
+									'Reach customers on other sales channels',
+									'woocommerce'
+								) }
+							</FlexBlock>
+						</Flex>
+					</FlexItem>
+					<FlexItem>
+						<Flex>
+							<Icon icon={ megaphone } />
+							<FlexBlock>
+								{ __(
+									'Advertise with marketing campaigns',
+									'woocommerce'
+								) }
+							</FlexBlock>
+						</Flex>
+					</FlexItem>
+					<FlexItem>
+						<Flex>
+							<Icon
+								icon={
+									<img
+										src={ wooIconUrl }
+										alt={ __(
+											'WooCommerce logo',
+											'woocommerce'
+										) }
+										width="24"
+										height="24"
+									/>
+								}
+							/>
+							<FlexBlock>
+								{ __( 'Built by WooCommerce', 'woocommerce' ) }
+							</FlexBlock>
+						</Flex>
+					</FlexItem>
+				</Flex>
+				{ ( showCreateCampaignButton || showAddChannelsButton ) && (
+					<Flex
+						className="woocommerce-marketing-introduction-banner-buttons"
+						justify="flex-start"
+					>
+						{ showCreateCampaignButton && (
+							<Button
+								variant="primary"
+								onClick={ () => {
+									setOpen( true );
+								} }
+							>
+								{ __( 'Create a campaign', 'woocommerce' ) }
+							</Button>
+						) }
+						{ showAddChannelsButton && (
+							<Button
+								variant="secondary"
+								onClick={ onAddChannels }
+							>
+								{ __( 'Add channels', 'woocommerce' ) }
+							</Button>
+						) }
+					</Flex>
+				) }
+				{ open && (
+					<CreateNewCampaignModal
+						onRequestClose={ () => setOpen( false ) }
+					/>
+				) }
+			</div>
+			<div className="woocommerce-marketing-introduction-banner-illustration">
+				<Button
+					isSmall
+					className="woocommerce-marketing-introduction-banner-close-button"
+					onClick={ onDismiss }
+				>
+					<Icon icon={ closeSmall } />
+				</Button>
+				<img
+					src={
+						showCreateCampaignButton || showAddChannelsButton
+							? illustrationLargeUrl
+							: illustrationUrl
```

Comment:
Yeah I thought about that, but the problem is with the gradient background in the SVG files. I think it would be better if we have just the product image in the SVG, and have the gradient background applied onto HTML div via CSS. I'll leave it as a future enhancement PR.

Reply:
>> I think it would be better if we have just the product image in the SVG, and have the gradient background applied onto HTML div via CSS.
>
> This could be done by these steps: [...]

Thanks @eason9487 for the detailed steps! You could have committed and contributed the code. 😂 I made some minor changes (done in https://github.com/woocommerce/woocommerce/pull/37110/commits/4e38969ce7d2f3b1246573fac46c4eb7b4d3b9af) and this is how it looks, which I think is acceptable and not much difference with the design:

![image](https://user-images.githubusercontent.com/417342/226312643-2b758ed5-8338-4918-9919-7b41ffd3dcd0.png)

I chose to use `background: center / contain no-repeat;` because I don't want the image to be stretched by `background: center / cover no-repeat;`, and I didn't use the `&--smaller` rule because I think that is not needed.